<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicFileInformationWidget</class>
 <widget class="QWidget" name="MusicFileInformationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>歌曲信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>475</width>
     <height>527</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>歌曲信息</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>457</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>54</y>
     <width>375</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>44</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>文件信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>74</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePathEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>74</y>
     <width>300</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>104</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>文件大小:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileSizeEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>104</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>74</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileFormatEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>104</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>104</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>格式:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>134</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>歌手信息:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileArtistEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>134</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileTitleEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>134</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>134</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>文件标题:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>164</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>专辑信息:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileAlbumEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>164</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>164</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>长度:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileTimeEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>164</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>194</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>流派:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileGenreEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>194</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>259</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>采样速率:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>294</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>轨段编号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TrackNumEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>294</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>294</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>通道:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SamplingRateEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>259</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ChannelEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>294</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileYearEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>194</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>194</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>年份:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>259</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>比特率:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BitrateEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>259</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>227</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>其他信息</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>237</y>
     <width>375</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>475</width>
     <height>537</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>495</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>495</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
  <widget class="MusicClickedLabel" name="advanceLabel">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>530</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;高级&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="pixmapLabel">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>405</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:Rerd</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>解码器:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openPixButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>选择图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>445</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>图片大小:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pixmapSizeLabel">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>445</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="decoderLabel">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>410</y>
     <width>210</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>328</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>音质:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qualityEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>328</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePixButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>490</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>删除图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePixButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>导出图片</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>line_2</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>filePathEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>fileSizeEdit</zorder>
  <zorder>viewButton</zorder>
  <zorder>fileFormatEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>fileArtistEdit</zorder>
  <zorder>fileTitleEdit</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>fileAlbumEdit</zorder>
  <zorder>label_8</zorder>
  <zorder>fileTimeEdit</zorder>
  <zorder>label_9</zorder>
  <zorder>fileGenreEdit</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>TrackNumEdit</zorder>
  <zorder>label_12</zorder>
  <zorder>SamplingRateEdit</zorder>
  <zorder>ChannelEdit</zorder>
  <zorder>fileYearEdit</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>BitrateEdit</zorder>
  <zorder>label_15</zorder>
  <zorder>line_3</zorder>
  <zorder>advanceLabel</zorder>
  <zorder>pixmapLabel</zorder>
  <zorder>openPixButton</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>pixmapSizeLabel</zorder>
  <zorder>decoderLabel</zorder>
  <zorder>label_16</zorder>
  <zorder>qualityEdit</zorder>
  <zorder>deletePixButton</zorder>
  <zorder>savePixButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicClickedLabel</class>
   <extends>QLabel</extends>
   <header>musicclickedlabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
